modules/fedimint-lnv2-common/src/config.rs

```diff
@@ -107,3 +108,51 @@ impl Default for FeeConsensus {
         }
     }
 }
+
+#[allow(dead_code)]
+fn migrate_config_consensus(
+    config: fedimint_ln_common::config::LightningConfigConsensus,
+    peer_count: u16,
+) -> LightningConfigConsensus {
+    LightningConfigConsensus {
+        tpe_agg_pk: AggregatePublicKey(config.threshold_pub_keys.public_key().0.to_affine()),
+        tpe_pks: (0..peer_count)
```

Comment:
Didn't we talk about not wanting this redundancy anymore? You proposed saving the public polynomial instead iirc and deriving both shares and the aggregated key from that on startup.